README.md

```diff
@@ -42,7 +42,7 @@ Clone the repository, change to the directory containing the repository.
 $ pip install -r requirements.txt
```

Comment:
Would it also be worth a few lines here for installing Python/Pip? 

I don't actively develop (let alone in Python), so `pip` is something I needed to go get before I was able to utilize. 

Happy to put this within its own PR, but I felt it might fit here, too.